src/components/attendee/VideoComms/internal/TwilioTrackDisplay.tsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useRef } from "react";
+import Twilio from "twilio-video";
+
+import styles from "./scss/TwilioTrackDisplay.module.scss";
+
+interface TwilioTrackDisplayProps {
+  track: Twilio.VideoTrack;
+  isMirrored: boolean;
+}
+
+/**
+ * This low level component exists to do the display of a track from Twilio.
+ * This component should not be used directly and only be components within
+ * the VideoComms folder.
+ */
+export const TwilioTrackDisplay: React.FC<TwilioTrackDisplayProps> = ({
+  track,
+  isMirrored,
+}) => {
+  const classNames = isMirrored ? styles.mirrored : "";
```

Comment:
maybe use our basic approach with using `classNames` and have it add `[styles.mirrored]: isMirrored` that way?
would look like this `classNames("", {[styles.mirrored]: isMirrored})`

Reply:
Resolved in 03a5d4919